prow/config.yaml

```diff
@@ -203,10 +203,6 @@ presubmits:
           value: true
         - name: GOOGLE_APPLICATION_CREDENTIALS
           value: /etc/service-account/service-account.json
-        - name: JENKINS_GCE_SSH_PRIVATE_KEY_FILE
-          value: /etc/ssh-key-secret/ssh-private
-        - name: JENKINS_GCE_SSH_PUBLIC_KEY_FILE
-          value: /etc/ssh-key-secret/ssh-public
```

Comment:
also the volumeMounts and volumes below

Reply:
We still need the docker in docker support to run the unit tests.

Reply:
I mean the ssh volume

Reply:
Ah got it. Fixed